frontend/src/components/App/Layout.tsx

```diff
@@ -109,6 +110,9 @@ export default function Layout({}: LayoutProps) {
    * indexDB and then sends the backend to parse it and then updates the parsed value into redux
    * store on an interval.
    * */
+  // DETAIL DRAWER MODE
```

Comment:
`// DETAIL DRAWER MODE` comment is not necessary and this new line is inserted between the useEffect and the comment related to useEffect, please move this line or move the comment above

Reply:
done!